vnext/PropertySheets/React.Cpp.props

```diff
@@ -18,14 +18,25 @@
 
   <PropertyGroup>
     <USE_HERMES Condition="'$(USE_HERMES)' == ''">false</USE_HERMES>
+    <HERMES_Version Condition="'$(HERMES_Version)' == ''">0.1.6</HERMES_Version>
```

Comment:
Is Hermes to be used throughout all RNW?
If not, I suggest creating a Hermes-specific `props` file and either:
1. Include it right after `React.Cpp.props` in the appropriate VCXPROJ files.
2. Wrap `React.Cpp.props` and include it in the appropriate VCXPROJ files.

Reply:
Julio.. thanks for the suggestion.. I'll do it in a subsequent change..